salt/metalk8s/saltmaster/minion_configured.sls

```diff
@@ -0,0 +1,31 @@
+{% set saltminion_version = '2018.3.3'%}
+
+
+Configure saltstack repo:
+  pkgrepo.managed:
+    - humanname: SaltStack repo for RHEL/CentOS \$releasever
+    - baseurl: https://repo.saltstack.com/yum/redhat/\$releasever/\$basearch/archive/{{ saltminion_version }}
+    - gpgcheck: 1
+    - gpgkey: https://repo.saltstack.com/yum/redhat/\$releasever/\$basearch/archive/{{ saltminion_version }}/SALTSTACK-GPG-KEY.pub
+
+Install salt minion:
+  pkg.latest:
+    - name: salt-minion
```

Comment:
Since you set version variable even if you configure repo only for those version you should specify it also here version